### ISSUE TITLE
Too long meta data loading (test framework dependencies?)

### ISSUE THREAD

**Issue by ekudel** [JetBrains/intellij-platform-gradle-plugin#1665]
### What happened?

Whenever I run tests (from IntelliJ IDEA using Gradle), it takes about **10 minutes** to start, because Gradle loads "meta data" for all "test-framework" libraries (see the screenshot). The project is not big by the way. I'm attaching a part of Gradle's DEBUG output.

### Relevant log output or stack trace

```shell
2024-06-24T17:24:24.045+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://cache-redirector.jetbrains.com/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://dtahfujkndrht.cloudfront.net/packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:24.188+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@6b37486d5 pairs: {GET /packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: dtahfujkndrht.cloudfront.net}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:24.349+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@6dda56db11 pairs: {null: HTTP/1.1 307 Temporary Redirect}{Content-Length: 0}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:24:24 GMT}{Server: Kestrel}{Location: https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{X-Artifact-Caching-Proxy: ALWAYS_REDIRECT}{X-Cache: Miss from cloudfront}{Via: 1.1 c4596d40130fcd1e37f17f3220ae02b0.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: x7hn2HealJSbh3MikyZ4ZPwr9PTYo2xHRErjg0-XgQzVO7rt_G_qwg==}
2024-06-24T17:24:24.350+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://dtahfujkndrht.cloudfront.net/packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:24.350+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@6b37486d5 pairs: {GET /maven/p/ij/intellij-dependencies/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: packages.jetbrains.team}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:24.478+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@3cff0f2d8 pairs: {null: HTTP/1.1 404 Not Found}{Date: Mon, 24 Jun 2024 12:24:24 GMT}{Content-Length: 0}{Connection: keep-alive}{Vary: Origin}{X-Frame-Options: DENY}{Content-Security-Policy: frame-ancestors 'none'}{Referrer-Policy: no-referrer}
2024-06-24T17:24:24.623+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@619c78eb5 pairs: {GET /www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: cache-redirector.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:24.739+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@5e4f868d19 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Server: nginx}{Date: Mon, 24 Jun 2024 12:24:24 GMT}{Location: https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Expires: Mon, 24 Jun 2024 12:24:23 GMT}{Cache-Control: no-cache}{X-Debug-RemoteAddr: 15.158.1.246}{X-Debug-ForwardedFor: 82.215.102.41}{X-Debug-ProxyProtocolAddr: 15.158.1.246}{X-Debug-RealipRemoteAddr: 15.158.1.246}{X-Debug-RedirectFromAwsBuildfarm: 0}{X-Cache: Miss from cloudfront}{Via: 1.1 1041d30496690ec9e05d9d1a21a80876.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: vVb7YqpXVeclpb357bCCWUVn-XmJHwkTOSYgbQaI2sPDWiU3t_fHJg==}{X-Robots-Tag: none: noarchive}
2024-06-24T17:24:24.740+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:24.886+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@619c78eb5 pairs: {GET /intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: www.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:24.957+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@64db8cbc14 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:20:41 GMT}{Server: nginx}{Location: https://d2cico3c979uwg.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Strict-Transport-Security: max-age=31536000;}{X-Cache: Hit from cloudfront}{Via: 1.1 16beacaf8cf9058094c22f0750f306ee.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P1}{Alt-Svc: h3=":443"; ma=86400}{X-Amz-Cf-Id: CYkQfyEhBJIMFiU9tWrV4xDxiqzBUV6JA_SOrfDGgUfRPcjsRzq3Vg==}{Age: 224}
2024-06-24T17:24:24.959+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://d2cico3c979uwg.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:24.959+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@619c78eb5 pairs: {GET /com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: d2cico3c979uwg.cloudfront.net}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:25.034+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@6682fbec15 pairs: {null: HTTP/1.1 200 OK}{Content-Type: text/xml}{Content-Length: 9596}{Connection: keep-alive}{Last-Modified: Fri, 21 Jun 2024 07:38:29 GMT}{x-amz-server-side-encryption: AES256}{Accept-Ranges: bytes}{Server: AmazonS3}{Date: Mon, 24 Jun 2024 12:15:36 GMT}{ETag: "e444f51e6b0f0bfe2aefecb30def54a4"}{X-Cache: Hit from cloudfront}{Via: 1.1 8e83ffcea906745255dd96fad91bf456.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: IHCU1PsLykOgRJP5-ux1Xy1JXJ5YxfileGVPDwwsCM8vEZ0xWFKrCQ==}{Age: 530}
2024-06-24T17:24:25.182+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@35471bd65 pairs: {GET /www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: cache-redirector.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:25.295+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@d41dfae19 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Server: nginx}{Date: Mon, 24 Jun 2024 12:24:25 GMT}{Location: https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Expires: Mon, 24 Jun 2024 12:24:24 GMT}{Cache-Control: no-cache}{X-Debug-RemoteAddr: 15.158.1.210}{X-Debug-ForwardedFor: 82.215.102.41}{X-Debug-ProxyProtocolAddr: 15.158.1.210}{X-Debug-RealipRemoteAddr: 15.158.1.210}{X-Debug-RedirectFromAwsBuildfarm: 0}{X-Cache: Miss from cloudfront}{Via: 1.1 9ea29f496c2d2a04e880709bd543ad02.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: cimpPOBL9UmtygHUvPALTuLatEnHDIDJsu8smnrvcr_tNl_Qe4xKgw==}{X-Robots-Tag: none: noarchive}
2024-06-24T17:24:25.298+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:25.441+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@35471bd65 pairs: {GET /intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: www.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:25.512+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@633b443914 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:20:42 GMT}{Server: nginx}{Location: https://d2s4y8xcwt8bet.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Strict-Transport-Security: max-age=31536000;}{X-Cache: Hit from cloudfront}{Via: 1.1 1006b6fade1118dfc3284a26ac82fccc.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P1}{Alt-Svc: h3=":443"; ma=86400}{X-Amz-Cf-Id: AuGlT2bV8oCuE5I2SV6PQZFIkqmUUTtF-iU3RQvL0ePqJPkp40szSQ==}{Age: 223}
2024-06-24T17:24:25.513+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://d2s4y8xcwt8bet.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:25.514+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@35471bd65 pairs: {GET /com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: d2s4y8xcwt8bet.cloudfront.net}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:25.586+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@6e2f6a2316 pairs: {null: HTTP/1.1 200 OK}{Content-Type: text/xml}{Content-Length: 6055}{Connection: keep-alive}{Last-Modified: Fri, 21 Jun 2024 17:34:48 GMT}{x-amz-expiration: expiry-date="Mon, 30 Sep 2024 00:00:00 GMT", rule-id="Cleanup as per ADM-22593"}{x-amz-server-side-encryption: AES256}{Accept-Ranges: bytes}{Server: AmazonS3}{Date: Mon, 24 Jun 2024 05:54:40 GMT}{ETag: "75a4da14d842a01d2fa4e7f6c770f902"}{X-Cache: Hit from cloudfront}{Via: 1.1 8a7b11c8a73c9363e6dd587e2c39686e.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P3}{X-Amz-Cf-Id: cf2Pc3ceiBOjcoWaYTXAkUHGNYGi4Um5927B0X6HKxDGeVC7rUOQpQ==}{Age: 23386}
2024-06-24T17:24:25.735+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@792a47b45 pairs: {GET /plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: cache-redirector.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:25.851+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@4009de2419 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Server: nginx}{Date: Mon, 24 Jun 2024 12:24:25 GMT}{Location: https://dtahfujkndrht.cloudfront.net/plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Expires: Mon, 24 Jun 2024 12:24:24 GMT}{Cache-Control: no-cache}{X-Debug-RemoteAddr: 15.158.1.246}{X-Debug-ForwardedFor: 82.215.102.41}{X-Debug-ProxyProtocolAddr: 15.158.1.246}{X-Debug-RealipRemoteAddr: 15.158.1.246}{X-Debug-RedirectFromAwsBuildfarm: 0}{X-Cache: Miss from cloudfront}{Via: 1.1 9ea29f496c2d2a04e880709bd543ad02.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: VZMYnEGa5XhkGlPTLS7ao2hYqUZ-r5Q-z6NQZtGZB2ngsWtkuv6YMg==}{X-Robots-Tag: none: noarchive}
2024-06-24T17:24:25.853+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://cache-redirector.jetbrains.com/plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://dtahfujkndrht.cloudfront.net/plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:25.996+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@792a47b45 pairs: {GET /plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: dtahfujkndrht.cloudfront.net}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:26.154+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@3bbbab7311 pairs: {null: HTTP/1.1 307 Temporary Redirect}{Content-Length: 0}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:24:26 GMT}{Server: Kestrel}{Location: https://plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{X-Artifact-Caching-Proxy: ALWAYS_REDIRECT}{X-Cache: Miss from cloudfront}{Via: 1.1 f6d84a10259ceb3740deb03491fabcd6.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: VYEJsKkdi0NAOYGpTqVwEGIjP2cITvPYC2NTQGxAwYkt1O6IeXHWZA==}
2024-06-24T17:24:26.156+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://dtahfujkndrht.cloudfront.net/plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://plugins.jetbrains.com/maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:26.301+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@792a47b45 pairs: {GET /maven/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: plugins.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:26.421+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@1e71589d21 pairs: {null: HTTP/1.1 404 Not Found}{Content-Type: text/html;charset=UTF-8}{Transfer-Encoding: chunked}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:24:26 GMT}{Set-Cookie: AWSALB=vcWLqsengfvT25fZMdfaw8lvlxwfzjcRmFdAo4MbLBeDUAkD9xN43o8JXZOzsHQQBxY28bYE5TFPUDGEI9Ik+zbgPmeue6xXzMJ6MBlfQ7i2hvrDR2n5rIi5xecR; Expires=Mon, 01 Jul 2024 12:24:26 GMT; Path=/}{Set-Cookie: AWSALBCORS=vcWLqsengfvT25fZMdfaw8lvlxwfzjcRmFdAo4MbLBeDUAkD9xN43o8JXZOzsHQQBxY28bYE5TFPUDGEI9Ik+zbgPmeue6xXzMJ6MBlfQ7i2hvrDR2n5rIi5xecR; Expires=Mon, 01 Jul 2024 12:24:26 GMT; Path=/; SameSite=None; Secure}{X-Content-Type-Options: nosniff}{X-XSS-Protection: 1; mode=block}{Cache-Control: no-cache, no-store, max-age=0, must-revalidate}{Pragma: no-cache}{Expires: 0}{Strict-Transport-Security: max-age=31536000 ; includeSubDomains}{X-Frame-Options: SAMEORIGIN}{Content-Security-Policy: connect-src 'self' localhost:* http://localhost:* *.jetbrains.com *.google-analytics.com *.hotjar.com wss://*.hotjar.com app-lon02.marketo.com bam.nr-data.net bam-cell.nr-data.net *.algolia.net *.algolianet.com *.amazonaws.com *.analytics.google.com *.clarity.ms https://jetbrains.team/releases hub.jetbrains.com youtrack.jetbrains.com; default-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline' tagmanager.google.com www.googletagmanager.com www.google-analytics.com static.hotjar.com script.hotjar.com js-agent.newrelic.com bam.nr-data.net bam-cell.nr-data.net resources.jetbrains.com www.youtube.com static.ads-twitter.com snap.licdn.com connect.facebook.net *.marketo.net *.clarity.ms; font-src data: 'self' *.jetbrains.com themes.googleusercontent.com fonts.gstatic.com static.hotjar.com; img-src 'unsafe-inline' data: *; style-src 'unsafe-inline' 'self' *.jetbrains.com cloud.typography.com tagmanager.google.com fonts.googleapis.com; frame-src 'self' www.googletagmanager.com vars.hotjar.com www.youtube.com hub.jetbrains.com youtrack.jetbrains.com;}{Content-Language: en}{Vary: origin,accept-encoding}{X-Cache: Error from cloudfront}{Via: 1.1 2b90933536a16ec4a169497a31947a76.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: RDTf9wcOuQH_40qvzJSxdBU83OaEGAp2YikEPQNn9oLT_8tyqFA4PA==}
2024-06-24T17:24:26.570+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@3b0548ad5 pairs: {GET /com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: download.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:26.640+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@114a4e6e18 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Date: Mon, 24 Jun 2024 12:20:43 GMT}{Server: nginx}{Location: https://download-cdn.jetbrains.com/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Strict-Transport-Security: max-age=31536000; includeSubdomains;}{X-Frame-Options: DENY}{X-Content-Type-Options: nosniff}{X-XSS-Protection: 1; mode=block;}{X-GeoCode: UZ}{X-Cache: Hit from cloudfront}{Via: 1.1 26c2e0e41f83d4efa53728215e07322a.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P3}{Alt-Svc: h3=":443"; ma=86400}{X-Amz-Cf-Id: tKnrt_7R05HKPH-H_QoOz_5RbH5jA2AZymEtzi8ebZMVrfVst0NWQg==}{Age: 223}
2024-06-24T17:24:26.641+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://download.jetbrains.com/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://download-cdn.jetbrains.com/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:26.785+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@3b0548ad5 pairs: {GET /com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: download-cdn.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:27.051+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@44aba0e217 pairs: {null: HTTP/1.1 404 Not Found}{Content-Type: text/html}{Content-Length: 5922}{Connection: keep-alive}{x-amz-replication-status: COMPLETED}{Last-Modified: Thu, 15 Dec 2022 10:41:38 GMT}{x-amz-version-id: 1GaziBTpvHgaaHBxNWRRcqCZn3ROqiQU}{Accept-Ranges: bytes}{Server: AmazonS3}{Date: Mon, 24 Jun 2024 04:49:28 GMT}{ETag: "81f40bf8b8d9f1646c5de2834fe316aa"}{X-Cache: Error from cloudfront}{Via: 1.1 837bfbe95037e42cdc86bcbd263354ea.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P2}{Alt-Svc: h3=":443"; ma=86400}{X-Amz-Cf-Id: _3P15pn3jXZnTfyZnrdXPktd_RmAkajJGt85wyWe6H4VZQCWO6Y5dw==}{Age: 27430}
2024-06-24T17:24:27.199+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@1600d5055 pairs: {GET /intellij-jbr/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: cache-redirector.jetbrains.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2024-06-24T17:24:27.312+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@49ff6eb619 pairs: {null: HTTP/1.1 302 Moved Temporarily}{Content-Type: text/html}{Content-Length: 138}{Connection: keep-alive}{Server: nginx}{Date: Mon, 24 Jun 2024 12:24:27 GMT}{Location: https://d2xrhe97vsfxuc.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml}{Expires: Mon, 24 Jun 2024 12:24:26 GMT}{Cache-Control: no-cache}{X-Debug-RemoteAddr: 15.158.1.246}{X-Debug-ForwardedFor: 82.215.102.41}{X-Debug-ProxyProtocolAddr: 15.158.1.246}{X-Debug-RealipRemoteAddr: 15.158.1.246}{X-Debug-RedirectFromAwsBuildfarm: 0}{X-Cache: Miss from cloudfront}{Via: 1.1 a67d5c5659b214035494485fb8e7ed46.cloudfront.net (CloudFront)}{X-Amz-Cf-Pop: HEL51-P4}{X-Amz-Cf-Id: 15_w8jsWc7_O3I0Hw3jlBpMLYbzFj5A0GPqJJrAJXaLaCcsgFpJJlQ==}{X-Robots-Tag: none: noarchive}
2024-06-24T17:24:27.313+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] Redirected from https://cache-redirector.jetbrains.com/intellij-jbr/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml to https://d2xrhe97vsfxuc.cloudfront.net/com/jetbrains/intellij/platform/test-framework/maven-metadata.xml
2024-06-24T17:24:27.458+0500 [DEBUG] [sun.net.www.protocol.http.HttpURLConnection] sun.net.www.MessageHeader@1600d5055 pairs: {GET /com/jetbrains/intellij/platform/test-framework/maven-metadata.xml HTTP/1.1: null}{User-Agent: Java/17.0.5}{Host: d2xrhe97vsfxuc.cloudfront.net}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
```

### Steps to reproduce

1. Create a Gradle run configuration in the IDE with "test" command
2. Run it

### Gradle IntelliJ Plugin version

2.0.0-beta6

### Gradle version

8.3

### Operating System

None

**Comment by YannCebron** [comment]
Why Gradle 8.3 and not 8.8?
Please also upgrade to Beta7

**Comment by ekudel** [comment]
Thank you for the quick reply. Unfortunately using the new versions didn't help.
Also, I've noticed that the issue can be reproduced by running "buildPlugin" task too. If I don't have "testFramework(...)" dependencies, it works good, but tests can't run of course.

**Comment by ekudel** [comment]
@hsz Maybe there's some workaround? It's now really hard to develop...

**Comment by hsz** [comment]
Sorry for the inconvenience, @ekudel!

Given that some dependencies are resolved using the current IntelliJ Platform version (build number), the actual dependency version needs to be obtained as exact or close-to.
Because of that, we need to scan all possible versions from repositories defined in your project.

I have adjusted this process and started using Gradle's Resource Handler to better cache XML responses.
This will be available soon in `2.0.0-beta8`

Also, did you enable the configuration cache in your project? This helps a lot as well.
